packages/dev/core/src/Materials/greasedLinePluginMaterial.ts

```diff
@@ -602,27 +389,31 @@ export class GreasedLinePluginMaterial extends MaterialPluginBase implements IGr
                         grlDir = normalize( grlDir1 + grlDir2 );
                     }
                     vec4 grlNormal = vec4( -grlDir.y, grlDir.x, 0., 1. );
-                    #ifdef GREASED_LNE_RIGHT_HANDED_COORDINATE_SYSTEM
+                    #ifdef GREASED_LINE_RIGHT_HANDED_COORDINATE_SYSTEM
                         grlNormal.xy *= -.5 * grlWidth;
                     #else
                         grlNormal.xy *= .5 * grlWidth;
                     #endif
+
                     grlNormal *= grl_projection;
+
                     #ifdef GREASED_LINE_SIZE_ATTENUATION
                         grlNormal.xy *= grlFinalPosition.w;
                         grlNormal.xy /= ( vec4( grl_aspect_resolution_lineWidth.yz, 0., 1. ) * grl_projection ).xy;
                     #endif
+
                     grlFinalPosition.xy += grlNormal.xy * grlSide;
                     gl_Position = grlFinalPosition;
 
                     vPositionW = vec3(grlFinalPosition);
-
+                #else
+                    grlCounters = grl_counters;
+                #endif
                 `,
                 // eslint-disable-next-line @typescript-eslint/naming-convention
-                "!gl_Position\\=viewProjection\\*worldPos;": "//", // remove
+                "#ifndef GREASED_LINE_SIZE_ATTENUATION !gl_Position\\=viewProjection\\*worldPos; #endif  ": "//", // remove

```

Comment:
This should part of the original shader should be always removed (it doesn't do any harm because the gl_Position is reassigned in the GRL shader, it is removed just for optimalisation purposes). So I put back the original line w/o the ifndef. Thanks!